src/couch/src/couch_passwords.erl

```diff
@@ -51,7 +51,10 @@ hash_admin_password("pbkdf2", ClearPassword) ->
                                         Salt ,list_to_integer(Iterations)),
     ?l2b("-pbkdf2-" ++ ?b2l(DerivedKey) ++ ","
         ++ ?b2l(Salt) ++ ","
-        ++ Iterations).
+        ++ Iterations);
+hash_admin_password("bcrypt", ClearPassword) ->
+    Iterations = list_to_integer(config:get("couch_httpd_auth", "iterations", "12")),
```

Comment:
You can't have different defaults for different algorithms. Either be consistent with the existing default value (which I believe is 10) or propose a change with justification.

Reply:
Ho, I guess I should rename the parameter into `logRounds` as the real amount if iterations performed is actually 2^logRounds.
I'll change it.